### ISSUE TITLE
Differences between buildroot and noobs

### ISSUE THREAD

**Issue by lurch** [raspberrypi/noobs#8]
This isn't really an "issue" as such, just information that I think ought to be reported/recorded somewhere...

After a bit of an effort, I managed to determine that the version of buildroot used in noobs is from this (arbitrary) git commit http://git.buildroot.net/buildroot/commit/?id=e9b771283765e74792df6a05775383f6d4da2636 rather than the "released" 2013.05 version.
So I was then able to produce a diff between the "source" version of buildroot, and the version of buildroot that's included into noobs, which people may find useful (to see exactly what NOOBS changes).

Short version:  http://www.andrewscheller.co.uk/buildroot-noobs.short.diff
Long version:  http://www.andrewscheller.co.uk/buildroot-noobs.long.diff
(I tried posting these diffs inline, but github wouldn't let me - possibly because they're too long?)
